compliance_checker/cfutil.py

```diff
@@ -53,15 +53,25 @@
 }
 
 
-def get_unitless_standard_names(xml_tree, units):
+def get_dimensionless_standard_names(xml_tree, standard_name):
     '''
-    Returns True if the units are unitless. Unitless includes units that have
-    no units and units that are defined as '1'.
+    Returns True if the units for the associated standard name are unitless.
+    Unitless includes units that have no units and units that are defined as
+    constant units in the CF standard name table i.e. '1', or '1e-3'.
```

Comment:
Maybe use the term "dimensionless" here too, for consistency? "units that have no units" is not very informative.